tests/torchutils_test.py

```diff
@@ -134,6 +134,38 @@ def test_atleast_2d_many():
     assert t4.ndim == 2
 
 
+def test_maybe_add_batch_dim_to_size():
```

Comment:
This is a lot, and perhaps even helpful but we should really move these tests (far few of them) into a doctest, because doctests spare documentation when the tests is short (I think they are a great idea):
https://docs.python.org/3/library/doctest.html

Pytest has support: https://docs.pytest.org/en/latest/doctest.html

Reply:
Or we can shorten them with `pytest.parametrize`?